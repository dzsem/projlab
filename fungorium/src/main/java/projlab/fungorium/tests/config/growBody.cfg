# Grow Body
# ------------------------------------------
# Init state
# ------------------------------------------
# 0
ADD TECTON
# 1
ADD MUSHROOMSPORE 0 0
# 2
ADD MUSHROOMSPORE 0 0
# 3
ADD MUSHROOMSPORE 0 0
# 4
ADD MUSHROOMTHREAD 0 0

SET MUSHROOMTHREAD STATE 4 UNCUT GROWN