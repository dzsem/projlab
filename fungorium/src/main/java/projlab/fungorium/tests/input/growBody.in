# Grow Body
# ------------------------------------------
# Init state
# ------------------------------------------
# 0
ADD TECTON
# 1
ADD MUSHROOMSPORE 0 0
# 2
ADD MUSHROOMSPORE 0 0
# 3
ADD MUSHROOMSPORE 0 0
# 4
ADD MUSHROOMTHREAD 0 0
# Input
# ------------------------------------------
# 5
MUSHROOMTHREAD 4 GROWBODY
# Print states
# ------------------------------------------
LIST TECTON
LIST MUSHROOMTHREAD
LIST MUSHROOMBODY