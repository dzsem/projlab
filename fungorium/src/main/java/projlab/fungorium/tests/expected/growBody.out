TECTON 0 0 5 1 3
MUSHROOMTHREAD 4 0 0 0 UNCUT GROWN
MUSHROOMBODY 5 0 5 0 NORMAL 0